pom.xml

```diff
@@ -102,6 +102,13 @@ under the License.
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
       <version>${mavenVersion}</version>
+      <exclusions>
```

Comment:
this exclusion is actually not necessary anymore after upgrading to maven-core to 3.1.1